examples/browserify.js

```diff
@@ -1,7 +1,7 @@
 var d3 = require("d3"),
     cloud = require("../");
 
-var fill = d3.scale.category20();
+var fill = d3.scaleOrdinal(d3.schemeCategory20);
```

Comment:
Is this d3 v4? I had to use d3.scale.original to get it working with the current master